README.md

```diff
@@ -1,2 +1,153 @@
-# Hello-World
-This is my first Microverse Hello World repository 
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Hello Microverse\] ](#-hello-microverse-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is a project applying the knowledge on linters and collaborative pair programming.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[User will have access to all the projects and used technologies]**
+- **[User will be updated with the recent working updates]**
+- **[User will be able to contact and purchase merchandise]**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+By following below steps a new developer can make instant use of the project.
+
+Get a local copy up and running now:
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
```

Comment:
- [ ] Good job using this template as requested by the previous reviewer, but I would advise you to fill all these sections so that your readme would be professional and even a visitor can understand your project at a glance and know how to use it.

Reply:
He @Olamarx so the only thing left is to fill the places left in the readme file?